doc/backup.md

```diff
@@ -5,6 +5,8 @@ To backup a LXD instance different strategies are available.
 ## Full backup
 This requires that the whole `/var/lib/lxd` folder will be backuped up.
 Additionally, it is necessary to backup all storage pools as well.
+The filesystem used for backups must support symlinks.
```

Comment:
Hmm, I'm confused as to why this is even mentioned?

If you make a tarball of /var/lib/lxd then what the filesystem supports doesn't matter and if you copy the data across, you'll need a LOT more that just symlinks support (xattrs, acls, POSIX permissions, long paths, case sensitive, ...).